web-api/tests/recommendations.rs

```diff
@@ -122,311 +84,103 @@ macro_rules! assert_order {
-    assert_eq!(
-        error,
-        RecommendationsResponse::Error(PersonalizedDocumentsError::NotEnoughInteractions),
-    );
-
-    interact(client, personalization_url).await?;
-
-    let documents = send_assert_json::<RecommendationsResponse>(
-        client,
-        build_request(
-            client,
-            personalization_url,
-            published_after,
-            include_properties,
-            Endpoint::Recommendations,
-        )?,
-        StatusCode::OK,
-        false,
-    )
-    .await;
-    assert!(matches!(documents, RecommendationsResponse::Documents(_),));
-    let RecommendationsResponse::Documents(documents) = documents else {
-        unreachable!();
     };
-    assert(&documents);
-
-    let documents = send_assert_json::<RecommendationsResponse>(
-        client,
-        build_request(
-            client,
-            personalization_url,
-            published_after,
-            include_properties,
-            Endpoint::PersonalizedDocumentsPost,
-        )?,
-        StatusCode::OK,
-        true,
-    )
-    .await;
-    assert!(matches!(documents, RecommendationsResponse::Documents(_),));
-    let RecommendationsResponse::Documents(documents) = documents else {
-        unreachable!();
-    };
-    assert(&documents);
-
-    let documents = send_assert_json::<RecommendationsResponse>(
-        client,
-        build_request(
-            client,
-            personalization_url,
-            published_after,
-            include_properties,
-            Endpoint::PersonalizedDocumentsGet,
-        )?,
-        StatusCode::OK,
-        true,
-    )
-    .await;
-    assert!(matches!(documents, RecommendationsResponse::Documents(_),));
-    let RecommendationsResponse::Documents(documents) = documents else {
-        unreachable!();
-    };
-    assert(&documents);
-
-    Ok(())
-}
-
-#[test]
-fn test_personalization_all_dates() {
-    test_two_apps::<Ingestion, Personalization, _>(
-        UNCHANGED_CONFIG,
-        UNCHANGED_CONFIG,
-        |client, ingestion_url, personalization_url, _| async move {
-            ingest_with_dates(&client, &ingestion_url).await?;
-            personalize(&client, &personalization_url, None, None, |documents| {
-                assert_order!(
-                    documents,
-                    ["d8", "d6", "d1", "d5"],
-                    "unexpected personalized documents: {documents:?}"
-                )
-            })
-            .await?;
-
-            Ok(())
-        },
-    );
 }
 
 #[test]
-fn test_personalization_limited_dates() {
+fn test_full_personalization_with_inline_history() {
     test_two_apps::<Ingestion, Personalization, _>(
         UNCHANGED_CONFIG,
-        UNCHANGED_CONFIG,
-        |client, ingestion_url, personalization_url, _| async move {
-            ingest_with_dates(&client, &ingestion_url).await?;
-            personalize(
+        Some(toml! {
+            [semantic_search]
+            score_weights = [0.5, 0.5, 0.]
```

Comment:
Out of interest why did you set the weight of the search score to 0?

:wrench:  Additionally it should be:

```suggestion
            [personalization]
            score_weights = [0.5, 0.5, 0.]
```

Reply:
again I just copied this from the other test, didn't think too much about it.